content/articles/quarkus-configuration-profiles/index.md

```diff
@@ -0,0 +1,576 @@
+Working with Quarkus as a Reactive framework is easy and appealing to Java users. It comes with exciting features such as hot reloading, reactive program development capabilities, quick boot and reboot time, MicroProfile specification support, among many others. 
+
+In this article, one will be able to learn about Quarkus profiles. These are like a collection of settings that can be assigned a specific tag to access them quickly. But, of course, access depends on the current environment in use.
+
+### Table of Contents
+- [Key takeaways](#key-takeaways)
+- [Prerequisites](#prerequisites)
+- [What is Quarkus Configuration Profiles?](#what-is-quarkus-configuration-profiles)
+- [Checking and configuring application properties](#checking-and-configuring-application-properties)
+  - [Creating a new Quarkus project](#creating-a-new-quarkus-project)
+    - [Project structure](#project-structure)
+  - [Set up some configurations](#set-up-some-configurations)
+  - [Testing the configurations](#testing-the-configurations)
+- [Configuring the application using Quarkus configuration profiles](#configuring-the-application-using-quarkus-configuration-profiles)
+  - [Quarkus' Configuration Profile formats](#quarkus-configuration-profile-formats)
+  - [Using default configuration profiles](#using-default-configuration-profiles)
+  - [Using user defined configuration profile](#using-user-defined-configuration-profile)
+- [Setting up and using variables in different profiles](#setting-up-and-using-variables-in-different-profiles)
+  - [Setting up variables](#setting-up-variables)
+  - [Injecting the variables](#injecting-the-variables)
+  - [Testing the variables](#testing-the-variables)
+    - [Testing in the dev profile](#testing-in-the-dev-profile)
+    - [Testing in the prototype profile](#testing-in-the-prototype-profile)
+- [Configuration profiles in YAML files](#configuration-profiles-in-yaml-files)
+- [Conclusion](#conclusion)
+- [References](#references)
+
+### Key takeaways
+Some points to be taken in by the end of the article follow up include:
+- What is Quarkus Configuration Profiles
+- Importance of configuration profiles
+- Some ways of setting up configuration profiles in Quarkus. These include setting the profiles in the `application.properties` and 'YAML' files.
+- Easily navigating between the profiles set during the development stages.
+
```

Comment:
@justusmbuvi, this part should come after prerequisites.

Reply:
Got it :+1: